packages/components/src/navigator/navigator-screen/component.tsx

```diff
@@ -75,12 +74,16 @@ function UnconnectedNavigatorScreen(
 	const classes = useMemo(
 		() =>
 			cx(
-				css( {
-					// Ensures horizontal overflow is visually accessible.
-					overflowX: 'auto',
-					// In case the root has a height, it should not be exceeded.
-					maxHeight: '100%',
-				} ),
+				css`
+					/* Ensures horizontal overflow is visually accessible. */
+					overflow-x: auto;
+					/* In case the root has a height, it should not be exceeded. */
+					max-height: 100%;
+
+					&:focus {
+						outline: none;
+					}
```

Comment:
Focus indication must always be visible. Instead of resetting native outline, we should provide our own custom focus style.